src/sentry/eventstore/base.py

```diff
@@ -117,6 +92,11 @@ def get_events(
         """
         raise NotImplementedError
 
+    def get_unfetched_events(
+        self, filter, orderby=None, limit=100, offset=0, referrer="eventstore.get_unfetched_events"
+    ):
+        raise NotImplementedError
```

Comment:
Docstring here is specifically important to explain the difference with get_events